src/App.tsx

```diff
@@ -1,11 +1,34 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import { useEffect, useState } from 'react';
+import { getTodos } from './api/todos';
+import { TodoFilter } from './components/TodoFilter';
+import { TodoList } from './components/TodoList';
+import { Filter } from './enum/Filter';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { filteredTodos } from './utils/filter';
 
-const USER_ID = 0;
+const USER_ID = 6387;
 
 export const App: React.FC = () => {
+  const [error, setError] = useState(false);
```

Comment:
Use [right naming for boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) Fix it everywhere

```suggestion
  const [isError, setIsError] = useState(false);
```